### DISCUSSION TITLE
LEDs too dim

### DISCUSSION THREAD

**Discussion by illusive-c** [awawa-dev/HyperHDR#891] [General]
Here's my setup:
-HyperHDR and PiCap running on Homebrew LG C3 65" OLED
-WLED on an ESP32(Current limit disabled)
-40A power supply with beefy 12AWG wires (Voltage at the farthest point from power injection in the strip is 4.8V)
-SK6812 LEDs(60LEDs/m)

When I set the colour to white on WLED and turn the brightness all the way up, it gets eye-searingly bright. 

When I let the HyperHDR control the LEDs, the colours are accurate enough but the brightness is about 30% of WLED's full blast. 

I messed around with the Image Processing settings. Decreasing the Colour Saturation to 1.0 from 1.5 helps but then the colours get washed up. 

Is there a way to increase the brightness without distorting the colours? 

EDIT:I checked the live capture and it looks fine (i.e brightness looks normal)

**Comment by illusive-c** [comment]
Upon further investigation, I think the problem is with the HDR content. i.e HyperHDR would drive the LEDs to full 255 brightness only if it gets the HDR colourspace 100% brightness signal from the TV. However, so far I haven't been able to find any content that'd drive the brightness to that degree. 

That's just guesswork on my part thoguh, would be nice if someone could confirm.

**Reply by BaconToasts** [reply]
Same problem here. Would be nice if there was a way to increase the brightness output for the colors with the option of excluding/including white. Some content is too dark. Let me know if you find a solution.

**Reply by illusive-c** [reply]
I'm now 99% sure the issue is with the HDR vs SDR content. I was watching an HDR movie yesterday and I got the LEDs to work acceptably by cranking up the luminance and saturation to above 8. Some colours were still a little washed up but at least the ambient lighting was noticeable. 

Today I tested the setup with an "ambilight test" video on Youtube that wasn't HDR and all I got was almost completely white lighting. For SDR content I had to reduce luminance and saturation below 2 to actually get colours. 

I can see there's an "SDR/HDR" switch in PicCap but toggling it doesn't seem to do anything. 

I'll just leave the settings for HDR content for now and hope there's a fix coming.

**Reply by illusive-c** [reply]
I sorted the issue for the most part I think. There's a setting called "HDR to SDR tone mapping" under the Advanced/Network Setting (for some reason) that from what I understand it converts the incoming HDR image to SDR. With this toggle on, I can leave the Luminosity/Saturation and RGB Gamma values around 1 and get an acceptably bright image with both SDR and HDR content.
 
I have a feeling if I poked around the LUT tables a little, I can get things even better but I've been playing around with HyperHDR settings for almost a week now and I just gave up.

**Reply by NatyaSadella** [reply]
Yeah we need different image processing profiles for SDR and HDR. Awawa said he doesnt see a sense behind it but many users need this. You can have the settings in image processing tab perfect for HDR but then you watch something in SDR and its way way too bright. This little easy feature would help so much.
